sdk/lib/core/int.dart

```diff
@@ -117,6 +156,15 @@ abstract class int extends num {
   /// and zero-bits are shifted in as the new most significant bits.
   ///
   /// The [shiftAmount] must be non-negative.
+  ///
+  /// Example:
+  ///
+  /// ```dart
+  /// print((3 >>> 1).toRadixString(2)); // 0011 -> 0001
+  /// print((9 >>> 2).toRadixString(2)); // 1001 -> 0010
+  /// print(((-9) >>> 2).toRadixString(2));
```

Comment:
I don't know about this example. Anyway for web it is `111111111111111111111111111101`